<!-- -*- mode: XML -*- -->
<launch>
	<arg name="launcher" 	default="ardrone2_launch" />

	<node pkg="ardrone_autonomy" type="ardrone_driver" name="ardrone_autonomy" clear_params="true">
		<param name="tf_prefix" value="ardrone_autonomy" />  
		<param name="realtime_navdata" value="true" />
		<param name="realtime_video" value="true" />		
		<param name="looprate" value="50" /> <!-- Tunes the speed at which the ros loop runs, and thus, the rate at which navdata is published -->
	    <remap from="/cmd_vel" to="ardrone/cmd_vel"/>
	</node>

	<!-- <node pkg="rostopic" 				type="rostopic" 		name="navdataEcho" 						output="screen" args="echo /ardrone/navdata"  /> -->
	<node pkg="optitrack_controller"	type="killNodes"		name="killNodes_$(arg launcher)"		output="screen" required="true"/>
	<node pkg="optitrack_controller"	type="listenForKill"	name="shutdownListener_$(arg launcher)"	output="screen"	required="true"/>

</launch>

